core/src/main/java/io/questdb/griffin/engine/table/SelectedRecord.java

```diff
@@ -145,6 +146,11 @@ public Long256 getLong256B(int col) {
         return base.getLong256B(getColumnIndex(col));
     }
 
+    @Override
+    public long getLongIPv4(int col) {
```

Comment:
We no longer need this override, no? Also, `getLongIPv4()` override can be removed from `PageFrameMemoryRecord`.